src/styles/_page_sign_in_out.scss

```diff
@@ -6,7 +6,7 @@
 .express-sign-in {
   @include page-padding;
 
-  .row {
+  [class*="__wrapper"] {
```

Comment:
to get around situations like this I would use the `@content;` directive in the mixin as we discussed => if the wrapper mixin accepted content then we could reduce then we could do away with using the cascade at all here.

Alternatively the mixin could not accept content, but rather the values could be overridden immediately after the mixin => this is a classic sass mixin pattern